Dockerfile

```diff
@@ -28,12 +28,7 @@ RUN npm install -g typescript@3.9.7 \
 RUN curl https://sh.rustup.rs -sSf |  bash -s -- -y
 RUN echo 'source $HOME/.cargo/env' >> $HOME/.bashrc
 
-# Install all build dependencies
-RUN ./build_dev.sh
```

Comment:
Removed this from the Dockerfile because the contents of the current directory aren't visible to the container (and thus the Dockerfile) until _after_ this is done running.

Removed the start command because we haven't done `yarn install` yet. 

I think we need Cam's help here >.>